types/repo/ChannelData.ts

```diff
@@ -68,4 +68,4 @@ export interface Command {
   set: string;
 }
 
-export type ChannelType = 'PM' | 'ANON_PM' | 'GROUP' | 'TOPIC';
+export type ChannelType = 'PM' | 'ANON_PM' | 'GROUP' | 'TOPIC' | 'ANON_TOPIC';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a new type 'ANON_TOPIC' to the ChannelType enum seems fine as long as it is handled properly in all places where this enum is used. Make sure that all switch-case or if-else blocks that use this enum are updated to handle this new type.
```diff
- export type ChannelType = 'PM' | 'ANON_PM' | 'GROUP' | 'TOPIC';
+ export type ChannelType = 'PM' | 'ANON_PM' | 'GROUP' | 'TOPIC' | 'ANON_TOPIC';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->